### DISCUSSION TITLE
No Logs in KIBANA- NETWORK- NTLM

### DISCUSSION THREAD

**Discussion by JAEP2** [Security-Onion-Solutions/securityonion#3999] [Unsupported Versions]
Hello,
In all my tests, in KIBANA- Network - NTLM doesn't show any Log.

Clear install with a Sensor and Managed Search.

All the other Panels Show logs correctly.

![2021-04-22 12_16_24-Security Onion - NTLM - Elastic](https://user-images.githubusercontent.com/31960081/115698166-9d892500-a364-11eb-9d1d-9a6d7d34ecfe.png)

Any sugestion to show the NTLM infomation in Kibana panel?

The network have NTLM traffic.

Thanks in advanced any help will be apreciated.

Kind regards

**Comment by weslambert** [comment]
Do you receive results if you search via Discover or Hunt, with something like the following?

`event.dataset:ntlm`

**Comment by JAEP2** [comment]
Hi Wes,

Thanks a lot for your reply

No results with event.dataset:ntlm on Hunt or Discover.

Kind Regards

**Reply by weslambert** [reply]
Is there an `ntlm.log` in `/nsm/zeek/logs/current/` on your sensor?

**Comment by JAEP2** [comment]
Hi Wes,

No ntlm.log in /nsm/zeek/logs/current/  , i will check if ntlm is disabled now on the network.
Thanks a lot

KInd regards
